scylla/src/client/session.rs

```diff
@@ -96,6 +94,21 @@ impl std::fmt::Debug for Session {
     }
 }
 
+#[cfg(feature = "__tls")]
+#[derive(Clone)] // Cheaply clonable - reference counted.
+#[non_exhaustive]
+pub enum TlsContext {
+    #[cfg(feature = "openssl-010")]
+    OpenSsl010(openssl::ssl::SslContext),
+}
```

Comment:
As I said during in-person conversation, I'd really want to remove this `__tls` feature so that `TlsContext` is not conditional - only its variants would be.